react/src/lib/Alert/examples/Default.js

```diff
@@ -1,36 +1,41 @@
 import React from 'react';
 import {
+  Alert,
   AlertContainer,
   Button
 } from '@momentum-ui/react';
 export default class AlertDefault extends React.PureComponent {
   state = {
-    alertMessage: 'Who\'s awesome?  You are!'
+    alertMessage: 'Who\'s awesome?  You are!',
```

Comment:
not using state.alertMessage in the examples anymore

Reply:
This was an artifact from when all the examples were in one file and the original author didn't want to write the same message.